### PR TITLE
Stop constantly rebuilding DHCP services.

### LINKED ISSUE #711: We're setting up networks too much
```
etcdctl get --prefix=True /sf/event/network/1b1c023b-87b0-4ec1-b4ab-23191266f9c5 | grep operation | sort | uniq -c | sort -n
      1     "operation": "allocate address",
      1     "operation": "db record creation",
      2     "operation": "create netns",
      2     "operation": "create physical veth",
      2     "operation": "create router veth",
      2     "operation": "deploy",
      3     "operation": "add mesh elements",
      3     "operation": "state changed",
      4     "operation": "create vxlan bridge",
      4     "operation": "create vxlan interface",
  21951     "operation": "network node",
  43902     "operation": "enable nat",
  43902     "operation": "enable virtual routing",
 109577     "operation": "update dhcp",
```

That's an allegedly idle network.

### PR DESCRIPTION
**Pull request by mikalstill**
Fixes #711.

### CODE REVIEW

**Comment by mandoonandy** [comment]
What happened to the unit tests?

File: deploy/shakenfist_ci/tests/test_floating_ips.py [hunk 1]
```diff
@@ -60,19 +60,18 @@ def test_simple(self):
...
             attempts += 1
             try:
-                r = requests.request(
-                    'GET', 'http://%s/' % ifaces[0]['floating'])
-                if r.status_code == 200 and r.text.find('Floating IPs work!') != -1:
-                    return
+                console = base.LoggingSocket(
+                    inst['node'], inst['console_port'])
+                out = console.execute('cat /var/www/html/index.html')
+                if out.find('Floating IPs work!'):
```
Comment (mandoonandy) [inline deploy/shakenfist_ci/tests/test_floating_ips.py:70]: should this check for != -1?

**Review by mandoonandy — CHANGES_REQUESTED** [review]

File: deploy/shakenfist_ci/tests/test_floating_ips.py (hunk 1 above)
Comment (mikalstill) [reply deploy/shakenfist_ci/tests/test_floating_ips.py]: Fair point.

**Review by mandoonandy — APPROVED** [review]
LGTM

**Pull request merged**